draft-rescorla-tls-rfc8446-bis.md

```diff
@@ -1330,6 +1330,8 @@ legacy_version:
   a legacy_version of 0x0303 and a supported_versions extension
   present with 0x0304 as the highest version indicated therein.
   (See {{backward-compatibility}} for details about backward compatibility.)
+  A client which receives a legacy_version value not equal to 0x0303 MUST abort
+  the handshake with an "illegal_parameter" alert.
```

Comment:
This seems to be missing legacy_compression_method?

Reply:
That’s already in the text for compression_method below, so I didn’t add it here.